docs/embedded/tutorials/migrate-abs-to-spe.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Sample App to Migrate from Azure Blob Storage container to SharePoint Embedded container
+description: Tutorial in how to migrate from Azure Blob Storage container to SharePoint Embedded container
+ms.date: 07/31/2024
+ms.localizationpriority: high
+---
+
+# Tutorial For Migrating Content From ABS To SPE
+
+## Purpose
+
+This tutorial will guide you through migrating content from Azure Blob Storage (ABS) to SharePoint Embedded (SPE). This is particularly useful for customers who have 500 docs in the blob storage container.
+
+### Prerequisites
+
+1. M365 subscription
+1. Azure account
+1. .NET Core SDK [version 8.0.303](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
+1. Dotnet environment to run sample app
+    - It can be run Windows, Linux and MacOs
+1. SharePoint Embedded container
+    - For more information on how to set up a [SPE container](https://aka.ms/start-spe)
+1. Azure Blob Storage container
+    - For more information on how to set up an [ABS container](../../../../azure/storage/blobs/storage-blobs-introduction)
+
+### Packages
+1. Microsoft Graph SDK (version 5.56.0)
```

Comment:
so sounds like Graph SDK is the main operational piece here, and apps can just call the SDK to deal with migration? Is that the idea? Under the hood, is the SDK calling some Graph APIs to deal with migration?